packages/terra-paginator/tests/wdio/paginator-spec.js

```diff
@@ -148,5 +148,15 @@ Terra.describeViewports('Controlled Paginator', ['large'], () => {
       $('#total-count-button').click();
       Terra.validates.element('custom page label with total count');
     });
+
+    it('should display custom page count with custom page label when hidePageCount is set to true', () => {
+      $('#hide-count-button').click();
+      Terra.validates.element('custom page label with custom page count');
+    });
+
+    it('should display progressive paginator with default page count when page label is not provided', () => {
+      $('#clear-page-lable-button').click();
+      Terra.validates.element('paginator with default page count');
+    });
```

Comment:
Other than wdio tests, could we also add some jest tests to reflect this change?

Reply:
Initially I thought of JEST but I was able to think of any value or additional validation that Jest can bring in this scenario as the change made here is more visually oriented. Please share your inputs on how Jest can add extra validation will include them if they really required.